cmd/context/create.go

```diff
@@ -279,6 +283,36 @@ func (c *ContextCommand) initOktetoContext(ctx context.Context, ctxOptions *Cont
 	return nil
 }
 
+// updateDynamicTokenForNamespace retrieves a dinamic token for the given userContext and updates Credentials.Token
+// if error while retrieving the dynamic token or flag OKTETO_USE_STATIC_KUBETOKEN is true, value is not updated
+// static token is fallback
+func updateDynamicTokenForNamespace(okClientProvider types.OktetoClientProvider, userContext *types.UserContext) {
```

Comment:
non-blocking suggestion:

```suggestion
func getAndUseDynamicKubetoken(okClientProvider types.OktetoClientProvider, userContext *types.UserContext) {
```

- the `update` part makes me think of a actual update of the token rather than "overriding" the static token
- the `ForNamespace` part can probably be omitted? Considering that the token is for a combination of Context and Namespace